Source/WebCore/page/Page.cpp

```diff
@@ -1106,7 +1106,15 @@ bool Page::shouldBuildInteractionRegions() const
 {
     return m_settings->interactionRegionsEnabled();
 }
-#endif
+
+void Page::enableBuildingInteractionRegions(bool enable)
+{
+    bool needsUpdate = enable && !shouldBuildInteractionRegions();
```

Comment:
I considered writing this as `enable != !shouldBuildInteractionRegions();`, but we really only need to rebuild the event regions when `enable == true`. I'm not sure if we need another check somewhere not to use the event regions if they are already built and enable gets set to false. In practice, I don't see it being an issue, but its certainly possible if this function is called after the event regions are already built